kale/loaddata/dataset_access.py

```diff
@@ -12,8 +12,7 @@ class DatasetAccess:
     of any dataset.
 
     Args:
-        n_classes: the number of classes (default=3).
-        ckernel_size: the size of the convolution kernel (default=5).
+        n_classes: the number of classes.
```

Comment:
Is data type (e.g. int, float) necessary here?

Reply:
Thanks. Done.